common/test/test_migrations/utils.py

```diff
@@ -0,0 +1,57 @@
+"""
+General class for Test migrations.Based off an implementation provided at
+https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
+"""
+# pylint: disable=redefined-outer-name
+
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.db.migrations.recorder import MigrationRecorder
+from django.test import TransactionTestCase
+from django.core.management import call_command
+
+
+class TestMigrations(TransactionTestCase):
+    """ Base class for testing migrations. """
+    migrate_from = None
+    migrate_to = None
+    app = None
+
+    def setUp(self):
+        super(TestMigrations, self).setUp()
+        assert self.migrate_from and self.migrate_to, \
+            "TestCase '{}' must define migrate_from and migrate_from properties".format(type(self).__name__)
+        assert self.app, "app must be define in the TestCase"
+        self.migrate_from = [(self.app, self.migrate_from)]
```

Comment:
yeah, wouldn't it be clearer if we didn't overwrite it? Then we wouldn't have monstrosities like
```python
call_command("migrate", self.app, self.migrate_from[0][1])
```
on L 55